src/main/groovy/de/dkfz/eilslabs/batcheuphoria/jobs/GenericJobInfo.groovy

```diff
@@ -31,6 +31,44 @@ class GenericJobInfo {
     BufferUnit memoryBufferUnit
     String queue
     String otherSettings
+    String user;
```

Comment:
WOOOHOOOOO :dancing_women: 
That's an awful lot of things. :smiley: 

Are they all generic enough?